deployment/release.sh

```diff
@@ -0,0 +1,9 @@
+#  version="USER INPUT"
+read -p "Enter version number: " version
```

Comment:
Also would be sweet if this fixed the version numbers in all he places we have to do it? :-)

Might be something for another ticket though!

Reply:
Yeah that would be cool - let me add a ticket for that

Reply:
For now, would it be worth just printing a reminder?  Something like:

Please update the version number to $ver in:
 - {Keanu locations}
 - {keanu starter}
 - {keanu docs stuff}